src/lib/geometry.f90

```diff
@@ -41,13 +41,17 @@ module geometry
   public element_connectivity_1d
   public element_connectivity_2d
   public inlist
+  public split_datacloud
+  public split_datacloudfromSTL
   public evaluate_ordering
   public get_final_real
+  public splitwords
```

Comment:
WE only need to make things public if they are used elsewhere or will have bindings.